examples.sh

```diff
@@ -0,0 +1,5 @@
+for example in ./examples/*.rs
+do 
+    FILE="$(basename "$example")"
+    cargo run --example "${FILE%.*}" -- $args
```

Comment:
nit, what is the purpose of this bash script? Given all the examples have hard coded values, I feel like you're treating the examples like end-to-end tests, which IMHO is not their goal 🤔